lib/warnings.js

```diff
@@ -54,4 +55,6 @@ warning.createDeprecation('FSTDEP019', 'reply.context property access is depreca
 
 warning.create('FastifyWarning', 'FSTWRN001', 'The %s schema for %s: %s is missing. This may indicate the schema is not well specified.', { unlimited: true })
 
+warning.create('FastifyWarning', 'FSTWRN002', 'The %s plugin you are trying to register has mixing async and callback style. This will return error in `fastify@5`', { unlimited: true })
```

Comment:
We should avoid "you"

```suggestion
warning.create('FastifyWarning', 'FSTWRN002', 'The %s plugin being registered mixes async and callback styles, which will result in an error in `fastify@5`', { unlimited: true })
```